client/src/components/profile/Profile.tsx

```diff
@@ -36,6 +41,39 @@ function a11yProps(index: number) {
 
 export const Profile = () => {
   const [currTab, setCurrTab] = useState(1)
+  // const [myEvents, setMyEvents] = useState<any[]>([])
+  const [isLoadingMyEvents, setIsLoadingMyEvents] = useState(false)
```

Comment:
can you split into two different components? single responsibility

Reply:
done